frontend/packages/data-portal/app/components/Run/ViewerPageTypes.ts

```diff
@@ -0,0 +1,33 @@
+export interface Annotation {
```

Comment:
All of this file is necessary? Those structure are derived from the graphQL queries and I don't think I saw the equivalent of that in the code for all the other queries. I understand it is on a typing point of view, but I would be afraid that it would lead to more errors in the future as if the query changes for any reasons, one needs to not forget to modify this also, otherwise the code will be correctly typed, but will fail at run time (unless I'm missing something)